cli/command/task.go

```diff
@@ -10,6 +13,162 @@ func NewTaskCommand(b *CommandBase) *TaskCommand {
 	return &TaskCommand{b}
 }
 
-func (e *TaskCommand) Command() cli.Command {
-	return cli.Command{}
+func (t *TaskCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "task",
+		Usage: "manage layer0 tasks",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new task",
+				Action:    t.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
+				Flags: []cli.Flag{
+					cli.IntFlag{
+						Name:  "copies",
+						Value: 1,
+						Usage: "number of copies of deploy to run (default: 1)",
+					},
+					cli.StringSliceFlag{
+						Name:  "env",
+						Usage: "environment variable override in format 'CONTAINER:VAR=VAL' (can be specified multiple times)",
+					},
+					cli.BoolFlag{
```

Comment:
the `--no-wait` flag is a global bool flag defined in main

Reply:
So it's not needed here then.

Reply:
correct